File: php/arrays.php
<?php

$start = microtime(true);

for ($i = 1; $i <= 1000; $i++) {
    $array = [];
    for ($j = 1; $j <= 1000; $j++) {
        $array[] = md5($j);
    }

    sort($array);
    $array = array_slice($array, 250, 500);
}

$end = microtime(true);
printf('%f sec.', $end - $start);

File: python/arrays.py
import time, hashlib

start = time.time()

prev = hashlib.md5(str(0).encode('utf-8')).hexdigest()
for i in range(1, 1000):
    array = []
    for j in range(1, 1000):
        array.append(hashlib.md5(str(j).encode('utf-8')).hexdigest())
    array.sort()

end = time.time()
print("{0} sec.".format(end - start))
